ultraplot/axes/base.py

```diff
@@ -1033,6 +1042,8 @@ def _add_colorbar(
         linewidth=None,
         edgefix=None,
         rasterized=None,
+        outline: Union[bool, None] = None,
+        labelrotation: Union[str, float] = "auto",
```

Comment:
Defaults needs to be handled via the rc mechanisms.

Reply:
You are right!